src/constructs/cloudwatch/alarm.test.ts

```diff
@@ -0,0 +1,65 @@
+import "@aws-cdk/assert/jest";
+import { ComparisonOperator } from "@aws-cdk/aws-cloudwatch";
+import { Runtime } from "@aws-cdk/aws-lambda";
+import { simpleGuStackForTesting } from "../../../test/utils";
+import { GuLambdaFunction } from "../lambda";
+import { GuAlarm } from "./alarm";
+
+describe("The GuAlarm class", () => {
+  it("should create a CloudWatch alarm", () => {
+    const stack = simpleGuStackForTesting();
+    const lambda = new GuLambdaFunction(stack, "lambda", {
+      code: { bucket: "bucket1", key: "folder/to/key" },
+      handler: "handler.ts",
+      runtime: Runtime.NODEJS_12_X,
+    });
+    new GuAlarm(stack, "alarm", {
+      alarmName: `Alarm in ${stack.stage}`,
```

Comment:
One for a future PR but I wonder if it would be useful to append stack, stage and app to the alarm name (or including the information in some other way) as part of the `GuAlarm`

Reply:
Good idea. Ideally we would add these as tags, but unfortunately [that's not currently possible](https://github.com/aws-cloudformation/aws-cloudformation-coverage-roadmap/issues/64), so perhaps we should put these into the name by default for now.